### PR TITLE
added ankr with reduced decimals, and usdc on polygon

### PR DESCRIPTION
**Pull request by morten-christiansen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coinify/currency",
-  "version": "1.3.26",
+  "version": "1.3.27",
```
Comment (mbnissen) [inline package.json:3]: remember to run `npm i` to update package-lock.json as well

File: src/currencies.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "ALL": { "name":"Albanian Lek",                         "decimals":2, "crypto":false },
   "AMD": { "name":"Armenian Dram",                        "decimals":2, "crypto":false },
   "ANG": { "name":"Netherlands Antillean Guilder",        "decimals":2, "crypto":false },
-  "ANKR": { "name":"Ankr Network ERC-20",                 "decimals":18, "crypto":true },
+  "ANKR": { "name":"Ankr Network ERC-20",                 "decimals":8, "crypto":true },
```
Comment (mbnissen) [inline src/currencies.json:9]: I assume this is safe to do and we don't have any data stored with 18 decimals

Comment (mbnissen) [reply src/currencies.json:9]: but please confirm :D

**Review by mbnissen — CHANGES_REQUESTED** [review]

File: src/currencies.json (hunk 2 above)
Comment (morten-christiansen) [reply src/currencies.json:9]: we are only just enablink ankr now, so i kind of assumed myself we didn't have any data on this. 
I have looked through swaps destination_currency and ankr is not there, are there any other places i need to look?

Comment (mbnissen) [reply src/currencies.json:9]: should be ok

**Review by mbnissen — APPROVED** [review]

**Pull request merged**